gitlab/v4/objects/statistics.py

```diff
@@ -19,6 +23,25 @@ class ProjectAdditionalStatisticsManager(GetWithoutIdMixin, RESTManager):
     _from_parent_attrs = {"project_id": "id"}
 
 
+class IssuesStatistics(RefreshMixin, RESTObject):
+    _id_attr = None
+
+
+class IssuesStatisticsManager(GetWithoutIdMixin, RESTManager):
+    _path = "/issues_statistics"
+    _obj_cls = IssuesStatistics
```

Comment:
When are `_optional_get_attrs` used? I see we have code for it in the CLI but only one class seems to use it ( `LicenseManager` )

Reply:
Realized I accidentally resolved this earlier. They are explicitly displayed with the CLI help if given, e.g.:

```shell
(.venv) nejc@nuc:~/repos/python-gitlab$ gitlab license get --help
usage: gitlab license get [-h] [--sudo SUDO] --key KEY [--project PROJECT] [--fullname FULLNAME]

optional arguments:
  -h, --help           show this help message and exit
  --sudo SUDO
  --key KEY
  --project PROJECT
  --fullname FULLNAME
```

Not sure how useful this is ATM, I'd wait for some CLI rework to consider this (it's another manual list to maintain).

Reply:
Thanks for the info.